convert/pom.xml

```diff
@@ -19,11 +19,6 @@
 
     <dependencies>
         <!--markdown libraries-->
-        <dependency>
```

Comment:
what is the purpose of this being removed? 

I understand that this is like being pulled as transitive dep in one of extensions, but to me keeping this one explicitly simply helps to highlight that this is the core part of markdown support, while everything else is an extra, that might be changed in future...

I propose to revert it back

Reply:
As you said it's pulled as transitive dep so why bother adding it.